cmd/scheduler/options/options.go

```diff
@@ -83,6 +87,10 @@ func New(origArgs []string) *Options {
 	fs.StringSliceVar(&opts.EtcdInitialPeers, "initial-cluster", []string{"dapr-scheduler-server-0=http://localhost:2380"}, "Initial etcd cluster peers")
 	fs.StringVar(&opts.EtcdDataDir, "etcd-data-dir", "./data", "Directory to store scheduler etcd data")
 	fs.StringSliceVar(&opts.EtcdClientPorts, "etcd-client-ports", []string{"dapr-scheduler-server-0=2379"}, "Ports for etcd client communication")
+	fs.StringSliceVar(&opts.EtcdClientHttpPorts, "etcd-client-http-ports", []string{""}, "Ports for etcd client http communication")
+	fs.Int64Var(&opts.EtcdSpaceQuota, "etcd-space-quota", 2*1024*1024*1024, "Space quota for etcd")
```

Comment:
For consistency, should we put the computed val here of `2147483648` instead of `2*1024*1024*1024`? 
```suggestion
	fs.Int64Var(&opts.EtcdSpaceQuota, "etcd-space-quota", 2147483648, "Space quota for etcd")
```